src/modules/ecdh/tests_impl.h

```diff
@@ -26,31 +26,19 @@ static int ecdh_hash_function_custom(unsigned char *output, const unsigned char
 
 static void test_ecdh_api(void) {
     /* Setup context that just counts errors */
```

Comment:
nit: Comment needs to be removed (or changed)

Reply:
fixed